cli/options/options_test.go

```diff
@@ -39,9 +39,9 @@ func TestGetTimeoutContext(t *testing.T) {
 		set := flag.NewFlagSet("flagSet", flag.ExitOnError)
 		ctx := cli.NewContext(cli.NewApp(), set, nil)
 		actualCtx, _ := GetTimeoutContext(ctx)
-		end := time.Now()
+		end := time.Now().Add(DefaultTimeout)
 		dl, _ := actualCtx.Deadline()
-		require.True(t, start.Before(dl) && dl.Before(end.Add(DefaultTimeout)))
+		require.True(t, start.Before(dl) && (dl.Before(end) || dl.Equal(end)))
```

Comment:
Looks weird, kinda against what the test should check for. So Windows timer doesn't have enough presicion?

Reply:
This check now is aimed to check that deadline is not earlier than estimated `end`, so I think it's still OK. In case of Windows deadline strictly equals to estimated `end`, not sure why it happens.